bin/compile

```diff
@@ -54,7 +54,8 @@ else
 fi
 
 APT_OPTIONS="-o debug::nolocking=true -o dir::cache=$APT_CACHE_DIR -o dir::state=$APT_STATE_DIR"
-APT_OPTIONS="$APT_OPTIONS -o dir::etc::sourcelist=$APT_SOURCES"
+# Override the use of /etc/apt/sources.list (sourcelist) and /etc/apt/sources.list.d/* (sourceparts).
+APT_OPTIONS="$APT_OPTIONS -o dir::etc::sourcelist=$APT_SOURCES -o dir::etc::sourceparts=-'"
```

Comment:
Why have `-'` as the value here? I would have expected this to be an empty or non-existent directory.

Reply:
Oh sorry the trailing `'` isn't meant to be there. The `-` is since that's what several SO / GitHub code references use. I tried to find a more authoritative guide as to what the supported path scheme was, however the docs are pretty lacking.

I'd be open to trying an empty string here and seeing whether that works too.

Reply:
Trying with the empty string results in `apt-get update` returning the man page (which doesn't cause the buildpack to fail early, due to #47).

I've switched it back to `-` (minus the `'` typo).

Reply:
So I found a reference to `/dev/null` in the APT source, here:
https://salsa.debian.org/apt-team/apt/blob/f15e090301e0744471d0fbf86ea0d494f6c08215/apt-pkg/sourcelist.cc#L313

...so I've switched it one last time to use that instead now 😆